bot/exts/code_jams/_cog.py

```diff
@@ -0,0 +1,236 @@
+import asyncio
+import csv
+import typing as t
+from collections import defaultdict
+
+import discord
+from botcore.utils.logging import get_logger
+from botcore.utils.members import get_or_fetch_member
+from discord import Colour, Embed, Guild, Member
+from discord.ext import commands
+
+from bot.bot import SirRobin
+from bot.constants import Emojis, Roles
+from bot.exts.code_jams import _channels
+from bot.services import send_to_paste_service
+
+log = get_logger(__name__)
+
+TEAM_LEADERS_COLOUR = 0x11806a
+DELETION_REACTION = "\U0001f4a5"
+
+
+class CodeJams(commands.Cog):
+    """Manages the code-jam related parts of our server."""
+
+    def __init__(self, bot: SirRobin):
+        self.bot = bot
+
+    @commands.group(aliases=("cj", "jam"))
+    @commands.has_any_role(Roles.admins)
```

Comment:
This will raise a `discord.ext.commands.errors.MissingAnyRole` if the invoker doesn't have this command.

We should catch this, so that it doesn't result in a stack trace in the bot.

Either with a cog-specific error handler, or a generic error handler cog.